config/kafka-instance-types-configuration.yaml

```diff
@@ -1,11 +1,18 @@
-# This configuration defines a list of supported Kafka instance types and their available sizes for KAS Fleet Manager. 
-# Modify this file to add additional Kafka instance types and sizes or change the default values listed below.
+# This configuration defines a list of supported Kafka instance types and their associated configuration for KAS Fleet Manager.
+# Modify this file to add additional Kafka instance types or change the default values listed below.
 # 
 # The following properties must be defined for each Kafka instance type:
 #   - id: Identifier for the Kafka instance type. Each instance type name should be unique.
 #   - display_name: human readable value of an instance type
+#   - [required] supported_billing_models: a list of available kafka billing models for the instance type. Cannot be empty
```

Comment:
doesn't have to be part of this pr but do we also want to follow this format for the other fields documented here?

Reply:
I think it would be a good idea

Reply:
We can do this in a separate PR, thanks! :)